lib/util/lifted/influx/httpd/handler.go

```diff
@@ -1247,7 +1247,7 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user meta2.
 		tsMultiplier = 1e9 * 3600
 	}
 
-	ctx := influx.GetStreamContext(body)
+	ctx := influx.GetStreamContext(body,h.Config.MaxLineSize)
```

Comment:
we need to add UT for GetStreamContext, because a new parameter added

Reply:
added